web-auth/src/components/Auth.svelte

```diff
@@ -11,18 +11,31 @@
 
   export let configParams: string;
   export let cloudClientIDs = "";
+  export let oktaName = "";
+  export let pingFedName = "";
+  export let disableForgotPassDomains = "";
 
   const cloudClientIDsArr = cloudClientIDs.split(",");
+  const disableForgotPassDomainsArr = disableForgotPassDomains.split(",");
 
-  // By default show the SignUp page
-  let isLoginPage = false;
+  // By default show the LogIn page
+  let isLoginPage = true;
   let errorText = "";
 
   let webAuth: WebAuth;
   const databaseConnection = "Username-Password-Authentication";
 
   $: loginOptions = LOGIN_OPTIONS;
 
+  $: loginOptions.forEach((option) => {
+    if (option.name === "Okta") {
+      option.connection = oktaName;
+    }
+    if (option.name === "Pingfed") {
+      option.connection = pingFedName;
+    }
+  });
+
```

Comment:
Can we just import the environment variables into `config.js` directly? (Btw, it'd be nice if that file were Typescript)